server/services/workAndSkillsPageService.ts

```diff
@@ -190,7 +236,10 @@ export default class WorkAndSkillsPageService {
     return govList
   }
 
-  public async getLearnerNeurodivergence(prisonerNumber: string, curiousApiClient: CuriousApiClient) {
+  private async getLearnerNeurodivergence(
```

Comment:
Corrected / strengthened the visibility of this function whilst I was here 👍 (nothing calls it outside of this class, so no need for it to be `public`)